### PR TITLE
fix(filter): #MED-319 fix some bugs on filters and add empty states

### PR DESCRIPTION
**Pull request by niko-amosse**
## Describe your changes

fix some bugs on filters and add empty states

image
image
image
...

## Checklist tests

check the operation of filters add display of empty state

## Issue ticket number and link

[MED-319](https://jira.support-ent.fr/browse/MED-319)

## Checklist before requesting a review (magic string, indentation, comment/documentation...)

- [ ] I have detailed the tests to do in my feature/fix in order to prevent consequents regressions (must specify in **Checklist tests**)
- [X] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequent feature, I have added thorough tests.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

File: frontend/src/components/filter-layout/utils.ts [hunk 1]
```diff
@@ -80,7 +80,7 @@ const filterByTypes = (resources: Resource[], types: string[]) => {
   }
   return resources.filter((resource) => {
     if (resource.source !== GAR || resource.is_textbook) {
-      return true;
+      return false;
```
Comment (niko-amosse) [inline frontend/src/components/filter-layout/utils.ts:83]: correctif bug sur le filtrage pour les ressources externes

File: frontend/src/components/empty-state/empty-state.tsx [hunk 2]
```diff
@@ -1,19 +1,21 @@
...
+  imgSource,
   title,
   description,
 }) => {
   const { t } = useTranslation();
   return (
     <div className="empty-state">
       <img
-        src="/mediacentre/public/img/empty-state.png"
+        src={imgSource ? imgSource : "/mediacentre/public/img/empty-state.png"}
```
Comment (Mattox0) [inline frontend/src/components/empty-state/empty-state.tsx:18]: à mon avis, met plus `/mediacentre/public/img/${imgSource ? imgSource : "empty-state.png}`
comme ça on a juste besoin de mettre le png aulieu de tout l'url je pense

File: frontend/src/components/infinite-scroll-list/InfiniteScrollList.tsx [hunk 3]
```diff
@@ -103,25 +102,19 @@ export const InfiniteScrollList: React.FC<InfiniteScrollListProps> = ({
         <LoadingScreen position={false} />
       ) : (
         <>
-          {visibleResources?.length ? (
```
Comment (Mattox0) [inline frontend/src/components/infinite-scroll-list/InfiniteScrollList.tsx:106]: pourquoi enlever cette sécu ?

File: frontend/src/components/filter-layout/FilterLayout.tsx [hunk 4]
```diff
@@ -95,68 +94,72 @@ export const FilterLayout: React.FC<FilterLayoutProps> = ({
...
-                  : "discipline"
+                resourcesInfosMap.types.length > 1 ? "types" : "type"
               }`,
             )}
           />
-        </div>
-      )}
+        )}
+        {isShowingThemes && (
+          <DropDown
```
Comment (ThomasSevagen) [inline frontend/src/components/filter-layout/FilterLayout.tsx:133]: factorisation

File: frontend/src/components/infinite-scroll-list/InfiniteScrollList.tsx (hunk 3 above)
Comment (niko-amosse) [reply frontend/src/components/infinite-scroll-list/InfiniteScrollList.tsx:106]: parce que l'empty state est géré dans chaque page vu que un diffèrent par page

File: frontend/src/components/empty-state/empty-state.tsx (hunk 2 above)
Comment (niko-amosse) [reply frontend/src/components/empty-state/empty-state.tsx]: c'est fait

**Review by ThomasSevagen — APPROVED** [review]

File: frontend/src/components/empty-state/empty-state.tsx [hunk 5]
```diff
@@ -1,19 +1,21 @@
...
+  imgSource,
   title,
   description,
 }) => {
   const { t } = useTranslation();
   return (
     <div className="empty-state">
       <img
-        src="/mediacentre/public/img/empty-state.png"
+        src={`mediacentre/public/img/${imgSource || "empty-state.png"}`}
```
Comment (Mattox0) [inline frontend/src/components/empty-state/empty-state.tsx:18]: t'as oublié le / du début

**Review by Mattox0 — APPROVED** [review]

**Pull request merged**